test/edu-benefits/utils/helpers.unit.spec.js

```diff
@@ -79,5 +79,14 @@ describe('edu helpers:', () => {
 
       expect(formatPartialDate(date)).to.be.undefined;
     });
+    it('should format a partial year', () => {
```

Comment:
Is this really able to validate dates with 2 digit years? When I test this locally it doesn't seem to work for me. A year of '07' is invalid.